src/functionalTest/resources/application-functional.yaml

```diff
@@ -26,4 +26,6 @@ exui.role.pui-case-manager: pui-case-manager
 resendInterval: 60
 
 # toggle for resendInvite
-resendInviteEnabled : false
\ No newline at end of file
+resendInviteEnabled : false
```

Comment:
resendInvite toggle should be turned on (as it is already on in prod)

Reply:
yes . i will enable

Reply:
done